src/pages/Service.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import { get } from 'lodash';
+import FormFeedback from 'components/FormFeedback';
+
+import s1 from '__tmpdata/service_1';
+import s2 from '__tmpdata/service_2';
+import s3 from '__tmpdata/service_3';
+
+const servicedata = {
+  1: s1,
+  2: s2,
+  3: s3
+}
+
+class Service extends Component {
+
+  render() {
+
+    const data = servicedata[this.props.match.params.id];
+
+    const topicId = get(data, "topic.id", null);
+    const topicName = get(data, "topic.name", null);
+    const title = get(data, "title", null);
+    const steps = get(data, "steps", null);
+    const body = get(data, "body", null);
+    const phone = get(data, "phone", null);
+    const email = get(data, "email", null);
+    const address = get(data, "address", null);
+    const hours = get(data, "hours", null);
+    const app = get(data, "app", null);
+    const related = get(data, "related", null);
+
+    return (
+      <div>
+
+        <div className="coa-section">
+          <a className="coa-page_breadcrumb" href={`/services/topic/${topicId}`}>{topicName}</a>
+          <h3 className="coa-page_title">{title}</h3>
+          { steps && (
+              <ol>{ steps.map((step) => { return <li>{step}</li> }) }</ol>
```

Comment:
this doesn't matter at all, but I like...
`<ol>{ steps.map((step) => <li>{step}</li>) }</ol>`